flink-connectors/flink-connector-kafka-base/src/main/java/org/apache/flink/streaming/connectors/kafka/internals/KafkaTopicPartitionStateSentinel.java

```diff
@@ -52,4 +52,7 @@
 	 */
 	public static final long GROUP_OFFSET = -915623761773L;
 
+	public static boolean isSentinel(long offset) {
+		return offset < 0;
```

Comment:
nit: this implementation could be a bit too broad. Could be a bit more specific by matching the static values in `KafkaTopicPartitionStateSentinel`.

Reply:
Kafka doesn't allow to commit any negative values